.github/workflows/python-package.yml

```diff
@@ -4,43 +4,51 @@
 name: Python package
 
 on:
-  push:
-    branches: [ master ]
   pull_request:
-    branches: [ master ]
+    types: [opened, synchronize, ready_for_review]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
-
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
Can you drop python 3.8? It's EOL https://devguide.python.org/versions/

You can update the readme to say

```
Python 3.9 or newer. Use of Python versions 3.8 or older are not supported.
```